exchange/adapter_map.go

```diff
@@ -183,7 +184,7 @@ func newAdapterMap(client *http.Client, cfg *config.Configuration, infos adapter
 
 	legacyBidders := map[openrtb_ext.BidderName]adapters.Adapter{
 		// TODO #267: Upgrade the Conversant adapter
-		openrtb_ext.BidderConversant: conversant.NewConversantAdapter(adapters.DefaultHTTPAdapterConfig, cfg.Adapters[string(openrtb_ext.BidderConversant)].Endpoint),
+		openrtb_ext.BidderConversant: conversant.NewConversantAdapter(adapters.DefaultHTTPAdapterConfig, cfg.Adapters[strings.ToLower(string(openrtb_ext.BidderConversant))].Endpoint),
```

Comment:
Sorry. I didn't notice this before. This needs to be removed. It's replaced by the `openrtb_ext.BidderConversant:` in the adapter map.